website/content/docs/commands/status.mdx

```diff
@@ -49,3 +49,21 @@ flags](/vault/docs/commands) included on all commands.
 - `-format` `(string: "table")` - Print the output in the given format. Valid
   formats are "table", "json", or "yaml". This can also be specified via the
   `VAULT_FORMAT` environment variable.
+
+By default, the output is displayed in "table" format.
+
+#### Output Fields
+
+1. The field for unseal nonce is displayed as `"n"` instead of `n` in yaml outputs. 
```

Comment:
Are you sure this is correct? Isn't that the field for "total shares", not "nonce"?

Reply:
Thanks for pointing this out. This should be "total shares" and not "nonce"